setup.py

```diff
@@ -30,16 +30,6 @@ def read_requirements(name):
     url="https://github.com/allegro/bigflow",
     packages=setuptools.find_packages(exclude=('test', 'e2e')),
     include_package_data=True,
-    data_files=[
```

Comment:
Why is it deleted?

Reply:
Because it is not needed nor used.
Now all *.txt files from bigflow are installed into $VIRTUAL_ENV/requirements
AFAIK nobody uses them.